### PR TITLE
RTL desktop header (bug 1145355)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/dev.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
     <meta name="msapplication-TileColor" content="#fff">
   </head>
 
-  <body class="overlayed" data-media="" data-local-html>
+  <body class="html-rtl overlayed" data-media="" data-local-html>
```
Comment (ngokevin) [inline src/dev.html]: probably just for debugging, but we can remove this now

**Comment by ngokevin** [comment]
i'd still vote for putting RTL styles in their own folder

File: src/dev.html (hunk 1 above)
Comment (spasovski) [inline src/dev.html]: oh yea good catch

**Comment by ngokevin** [comment]
I probably wouldn't give the current header the time of day given the nav will land before the RTL due-date.

**Comment by spasovski** [comment]
Old header went the way of the dino.

**Pull request closed without merging**